frontend/src/context/GameContextProvider.js

```diff
@@ -9,7 +9,10 @@ export default function GameContextProvider({ children }) {
 
   function signInGame(gameId, username) {
     return userSignIn(username)
-      .then((token) => signInGamePost(token, gameId))
+      .then((token) => {
+        console.log(username);
```

Comment:
```suggestion
```

Reply:
I am feeding a unique Id in the UsernamePasswordautehticationtoken in the jwt filter, because otherwise, someone could just get a token with the same name and  i would not be able to know which of the users it is. And i did not get the playerId and the usernem into the principal... I would be happy to avoid this